### PR TITLE
docs($markdown): add missing examples for line highlighting (fix #2473)

### LINKED ISSUE #2473: Missing examples for highlighting range of Lines in code blocks
## Feature request

[Line Highlighting in Code Blocks](http://localhost:8080/guide/markdown.html#line-highlighting-in-code-blocks)
Screen Shot 2020-07-01 at 00 07 59

There's only one example for this section.

However, it supports highlighting a range of lines, and even multiple ranges of lines!

In addition, import code highlighting works in the same way #2441

#### How should this be implemented in your opinion?

 Add more description for this feature. Provide examples for highlighting a range of lines and multiple ranges of lines.

#### Are you willing to work on this yourself?
Yep, but still it's Contribution Welcome!

### PR DESCRIPTION
**Pull request by diogotcorreia**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

I've kind of ran out of ideas for the code snippet, so if you have any suggestions feel free to comment down below 😃

### CODE REVIEW

**Comment by billyyyyy3320** [comment]
I'll update zh translation later.

File: packages/docs/docs/guide/markdown.md [hunk 1]
```diff
@@ -275,6 +275,46 @@ export default {
 }
 ```
 
+In addition to a single line, you can also specify multiple lines, line ranges, or both.
```
Comment (billyyyyy3320) [inline packages/docs/docs/guide/markdown.md:278]:
```suggestion
In addition to a single line, you can also specify multiple single lines, line ranges, or both.
```

File: packages/docs/docs/guide/markdown.md [hunk 2]
```diff
@@ -275,6 +275,46 @@ export default {
...
+
+- Line ranges: e.g. `{5-8}`, `{3-10}`, `{10-17}`
+- Multiple single lines: e.g. `{4,7,9}`
+- Line ranges and single lines: e.g. `{4,7-13,16,23-27,40}`
+
+**Input**
+
+````
+``` js{1,4,6-7}
+export default { // Highlighted
```
Comment (billyyyyy3320) [inline packages/docs/docs/guide/markdown.md:288]: When I was doing the translation, I was confused by `// Highlighted`. What does this stand for? Not only this line is highlighted.

Comment (diogotcorreia) [reply packages/docs/docs/guide/markdown.md:288]: I added that just as an indication that line would also be highlighted.

The other lines all have something saying they're highlighted (line 4 literally says that and lines 6 says lines 6 to 8), so I thought I'd add something there.

Comment (billyyyyy3320) [reply packages/docs/docs/guide/markdown.md:288]: I see. Thanks a lot!

**Pull request merged**